plugin/pkg/auth/authorizer/webhook/webhook.go

```diff
@@ -180,10 +186,10 @@ func (w *WebhookAuthorizer) Authorize(attr authorizer.Attributes) (err error) {
 		}
 	}
 	if r.Status.Allowed {
-		return nil
+		return true, nil
 	}
 	if r.Status.Reason != "" {
-		return errors.New(r.Status.Reason)
+		return false, errors.New(r.Status.Reason)
```

Comment:
I think we want a return type of `bool, string, error` to allow preserving this info